components/Icons/LikeIcon.tsx

```diff
@@ -1,18 +1,32 @@
 import * as React from 'react'
+import theme from '../../theme'
 interface SVGRProps {
   title?: string
   titleId?: string
+  filled?: boolean
 }
 
-function LikeIcon({ title, titleId, ...props }: React.SVGProps<SVGSVGElement> & SVGRProps) {
+function LikeIcon({
+  title,
+  titleId,
+  filled = false,
+  ...props
+}: React.SVGProps<SVGSVGElement> & SVGRProps) {
   return (
-    <svg width={16} height={14.119} viewBox="0 0 16 14.119" aria-labelledby={titleId} {...props}>
+    <svg
+      width={16}
+      height={14.119}
```

Comment:
Should have mentioned this when we were looking at this, but at least in the past fractional pixels produced inconsistent rendering and some extra work for the browser that you almost never need. The viewbox and anything that's measured in SVG's abstract units can be fractional without any worries, but width/height should be integers. I think 16x14 would be the way to go here

Reply:
Updated everything to match!